_security-analytics/api-tools/correlation-eng.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: default
+title: Correlation engine APIs
+parent: API tools
+nav_order: 55
+---
+
+
+# Correlation engine APIs
+
+Correlation engine APIs allow you to create new correlation rules, view findings and correlations within a certain span of time, and perform other tasks.
+
+
+## Create correlation rules between log types
+
+Used to create correlation rules.
```

Comment:
Please rephrase as a complete sentence naming the noun (what is used to do this?).